# Goal: XY coordinates
geometry_msgs/Point[] waypoints
---
# Result: success or error string
string message
float64 time_elapsed
---
# Feedback: remaining distance
float64 distance_remaining
float64 angle_remaining
int32 waypoints_remaining